### PR TITLE
Disable Version Increment -- Management Packages

### PR DESCRIPTION
**Pull request by scbedd**
Pending agreement that this doesn't fit in their workflow.

### CODE REVIEW

File: eng/versioning/version_shared.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 logging.getLogger().setLevel(logging.INFO)
 
 def path_excluded(path):
```
Comment (weshaggard) [inline eng/versioning/version_shared.py:36]: What all does this exclude? I suspect it disables more then just the version increment.

Comment (weshaggard) [reply eng/versioning/version_shared.py:36]: Nevermind I guess I should look at the script name. I only looked at the method name.

**Review by weshaggard — APPROVED** [review]
Change looks reasonable. I must say I was expecting a change somewhere in the yml files though but I suppose this will do the job as well. 

Of course this is assuming they agree they don't want this versioning strategy.

**Comment by scbedd** [comment]
They agreed. Disabling.

**Pull request merged**